ros_packages/datatypes/action/RecordAudio.action

```diff
@@ -0,0 +1,6 @@
+float32 max_silent_seconds_before # the amount of silent-seconds waited before recording is stopped, if user has not started talking yet
+float32 max_silent_seconds_after # the amount of silent-seconds waited before recording is stopped, if user has already started talking
+---
```

Comment:
this would definitely make sense. Since this is behavior was not part of the original voice-assistant, this needs to be discussed with the PO and can then be implemented in a follow-up story